components/LeagueCard/LeagueCard.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { cn } from '@/utils/utils';
+import Image from 'next/image';
+import placeholderImage from './placeholderImage.svg';
+
+interface ILeagueCardProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
Why is this extending React.HTMLAttributes<HTMLDivElement>? Decouple any Interfaces into their own files. There's only 1 in this file, so create ILeagueCard.ts with the 1 interface.

Reply:
In creating my own props for the LeagueCard component, which is shadcn/ui's Card component, I wanted to ensure it was following the same structure as shadcn/ui's. This is why I was extending the React.HTMLAttributes.